.github/CODEOWNERS

```diff
@@ -734,6 +734,9 @@
 # ServiceLabel: %Network - Private Link
 # ServiceOwners:                                                   @privlinksuppgithub
 
+# PRLabel: %Network - Network Cloud
+/sdk/networkcloud/                                                 @Azure/azure-sdk-write-networkcloud
```

Comment:
Team won't be recognized by the linter for 24 hours after creation.  Assuming that it has the necessary configuration, we'll just need to rerun that step tomorrow.